internal/host/host.go

```diff
@@ -319,6 +319,7 @@ func (m *Manager) RefreshStatus(ctx context.Context, h *models.Host, db *gorm.DB
 	if err != nil {
 		return err
 	}
+	m.reportValidationFailedMetrics(ctx, h, conditions)
```

Comment:
Wouldn't this spam our metrics?
Each failed validation will occur multiple times.
More ever, validations such as NTP, usually fail first, while NTP is synching, and after NTP completed synching the validation will not fail anymore.
With this, the clusters, that do not have NTP issues will show up in the metrics as if they have failed, no?

Reply:
+1 it will report metrics every 8 seconds for every cluster, maybe report only on changes? 
besides that most of the validations will fail until cluster have hosts.

Reply:
> Wouldn't this spam our metrics?
Depends on you definition of "spam metrics", it will expose 2 new metrics (the second will be in a later PR):
* `assisted_installer_host_validation_failed` 
* `assisted_installer_cluster_validation_failed`

In each case the type of the validation failure will be one of the labels.

For each validation that failed (and each time it does) we will get a new metric.

In the case you describe with NTP, if it failed 3 times and then succeed on all other checks then you will see:
`assisted_installer_host_validation_failed{clusterId=<cluster-id>,emailDomain="redhat.com",openshiftVersion="4.6"} = 3`

> With this, the clusters, that do not have NTP issues will show up in the metrics as if they have failed, no?
If NTP validation failed few time before succeeded then YES, otherwise NO.

Reply:
what do you mean 3 times? if it takes 10 minutes for the validation to pas you will see it 75 times no?